### PR TITLE
Simple Demo

### PR DESCRIPTION
**Pull request by christiangriset**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+all: run
+
+run: build
+	./docker-demo
+
+build: vendor
+	go build
+
+vendor:
+	godep save ./...
```
Comment (nathanleiby) [inline Makefile:14]: needed? there are no deps

Comment (nathanleiby) [inline Makefile:14]: but i see it's consistent with other demo so maybe easier to keep aligned

**Comment by nathanleiby** [comment]
again, small README describing usage would be great

**Comment by nathanleiby** [comment]
lgtm

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.5
+
+RUN mkdir -p /go/src/github.com/Clever/docker_demo
+ADD . /go/src/github.com/Clever/docker_demo
+
+WORKDIR /go/src/github.com/Clever/docker_demo
+
+RUN go install github.com/Clever/docker_demo
+RUN go build -o /usr/local/bin/stoked github.com/Clever/docker_demo
```
Comment (natebrennand) [inline Dockerfile]: s/stoked/docker_demo

**Comment by nathanleiby** [comment]
:+1:

**Comment by natebrennand** [comment]
@cgclever, can this be closed / merged? it's showing up in the "Open Pull Requests" email

**Pull request closed without merging**